build/mvn

```diff
@@ -48,11 +48,11 @@ install_app() {
     # check if we already have the tarball
     # check if we have curl installed
     # download application
-    [ ! -f "${local_tarball}" ] && [ -n "`which curl 2>/dev/null`" ] && \
+    [ ! -f "${local_tarball}" ] && command -v curl &>/dev/null && \
```

Comment:
Instead of `&>/dev/null`, which seems a little confusing, can't we use

```
if [ $(command -v curl) ];
```

to test for the existence of `curl`?

We don't have to redirect output if we do it this way, and it's much more readable. I suggest using this pattern throughout.